### DISCUSSION TITLE
Remove from navigation

### DISCUSSION THREAD

**Discussion by ufukseyithan** [Z3d0X/filament-fabricator#139] [Q&A] [answered]
How can I remove the PageResource? I only want to use the PageBuilder form field in my other forms. I also want the resource to be not registered in the navigation.

Moreover, I already had a PageResource of mine. And they do conflict with each other. Even though, in the config file the resource field is set to the correct PageResource.

**Comment by Z3d0X** [comment, marked as answer]
Set `null` for resource in the config file, this was fixed in the latest update
